### DISCUSSION TITLE
Is there a round button for e.g. for the call action in flex?

### DISCUSSION THREAD

**Discussion by thomaschaaf** [twilio-labs/paste#1960] [General] [answered]
I am looking for a button which is round e.g. for calling and hanging up a call.

**Comment by loreina** [comment]
Hi @thomaschaaf, could you share more about the use cases for a round button? And would you mind sharing a screenshot or link to your designs for more context? Thanks!

**Reply by thomaschaaf** [reply]
We're building a call history. A button in flex to indicate calling and hanging up is (currently) round (as shown on the right)

In our call history we would like to use the same language and have a button where people can click to call.

![Bildschirmfoto 2021-10-21 um 18 37 03](https://user-images.githubusercontent.com/893393/138320518-716e6470-5b83-49bb-a869-556e426e6f6f.png)

**Reply by loreina** [reply]
@thomaschaaf Thanks for sharing! We have it on our roadmap to create a round variant of our Button component. I've also created a ticket in our backlog for this. Our team will follow up on its progress, thanks!

**Comment by loreina** [comment, marked as answer]
👋 Hi @thomaschaaf, following up to share that we have [circle buttons](https://paste.twilio.design/components/button#circle-buttons) available in Paste.
